docs/docs/multi-dc.md

```diff
@@ -0,0 +1,176 @@
+Multi Datacenter
+================
+
+As of version 0.2.0 Cassandra on Mesos allows for providing configuration
+to allow multiple datacenters to be aware of one another.
+
+
+## How it works
+
+Cassandra on Mesos allows an operator to configure the framework to provide the Datacenter and rack level information
+to Cassandra upon starting each server.
+
+Cassandra on Mesos can be configured to query other instances of Cassandra on Mesos to provide seed sets across
+multiple clusters in order to allow each of the clusters to coordinate and replicate data.  Periodically the seeds
+resource in the http API will be queried for each datacenter, when the seeds are able to be resolved they will be appended
+to the seed set when each server iss started.
+
+
+## Requirements
+
+1. Each individual cluster will need to be able to route to the IP addresses of each of the other clusters.
+2. All clusters need to have the same cluster name
+3. Datacenters need to be defined before starting the framework. New datacenters can not yet be added after a cluster has initialized.
+4. Node sizes and counts should be the same across all clusters.  The framework does not enforce or validate this, but should be configured equivalently on all clusters.
```

Comment:
Change to: "Node sizes and counts must be the same across all clusters.  The framework does not enforce or validate this, but this must be configured equivalently on all clusters."